dotnet/src/Microsoft.AutoGen/Agents/Services/InProcess/InMemoryMessageQueue.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// InMemoryMessageQueue.cs
+
+using System.Threading.Channels;
```

Comment:
It's not used anywhere, shall we still keep it?

Reply:
removed, thanks